sirepo/job_driver/sbatch.py

```diff
@@ -182,12 +186,22 @@ def init_class():
     global cfg, _KNOWN_HOSTS
 
     cfg = pkconfig.init(
+        cores=(None, int, 'dev cores config'),
         host=pkconfig.Required(str, 'host name for slum controller'),
         host_key=pkconfig.Required(str, 'host key'),
-        cores=(None, int, 'dev cores config'),
         shifter_image=(None, str, 'needed if using Shifter'),
         sirepo_cmd=pkconfig.Required(str, 'how to run sirepo'),
         srdb_root=pkconfig.Required(_cfg_srdb_root, 'where to run job_agent, must include {sbatch_user}'),
+        supervisor_host=(
```

Comment:
This tuple could actually come from job.SUPERVISOR_URI_CFG_DECL or something like that so it can be used by all driver decls.

Reply:
Can you elaborate on this? I've changed it to be supervisor_uri in all drivers so each one of them can be overriden individually. They use job.DEFAULT_SUPERVISOR_URI as their default value.

Reply:
See 584358a8

Reply:
Of course, thanks.